### PR TITLE
Build v1.2.0

### PR DESCRIPTION
**Pull request by basnijholt**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #12

### CODE REVIEW

**Comment by BastianZim** [comment]
Sorry, I accidentally closed the wrong issue. You should be added to the maintainer team now. Somehow, the bot failed before.

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

**Comment by BastianZim** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -302,4 +333,5 @@ extra:
   recipe-maintainers:
     - basnijholt
     - BastianZim
+    - basnijholt
```
Comment (BastianZim) [inline recipe/meta.yaml]:
```suggestion
```

**Comment by basnijholt** [comment]
Thanks @BastianZim!

I plan to look at this again soon.

Something you might be interested in, I added `qsimcirc` here https://github.com/conda-forge/staged-recipes/pull/24504.

Would you like to be a recipe maintainer there too?

**Comment by BastianZim** [comment]
Thanks!

Sure, please feel free to add me, although I currently have limited capacity, unfortunately.

**Comment by basnijholt** [comment]
@BastianZim, I found the [problem](https://github.com/conda-forge/cirq-feedstock/pull/13/commits/4e0f1e04ccc940b95961065bc385d02a7133c353) :tada:

Unfortunately, it now turns out that the rigetti dependencies are unresolvable.

**Comment by basnijholt** [comment]
@conda-forge-admin, please rerender

**Comment by BastianZim** [comment]
> Unfortunately, it now turns out that the rigetti dependencies are unresolvable.

Hmm how does upstream solve that? Maybe it's a bug that we should file there then?

**Comment by basnijholt** [comment]
@BastianZim, I expect (hope 🤞) that this PR fixes this https://github.com/conda-forge/rpcq-feedstock/pull/40.

Otherwise, we should just disable the rigetti build for now.

I have also opened https://github.com/rigetti/rpcq/issues/160.

**Comment by basnijholt** [comment]
> Hmm how does upstream solve that?

Probably with pip this is not a problem because pip doesn't resolve version conflicts correctly.

**Comment by basnijholt** [comment]
OK I went to the bottom of this.

- [cirq requires  pyquil~=3.2.0](https://github.com/quantumlib/Cirq/blob/v1.2.0/cirq-rigetti/requirements.txt)
- [pyquil 3.2.0 requires rpcq = "^3.10.0" (same for pyquil=3.2.1)](https://github.com/rigetti/pyquil/blob/06dc9e5fa87fccda41344928af01535b24ca8c0a/pyproject.toml#L28)
- [rpcq 3.10.0 requires msgpack>=0.6,<1.0](https://github.com/rigetti/rpcq/blob/fd482a50e8f33240ef424b3768fed0ed57495116/setup.py#L54)


When creating a clean environment and running `pip install cirq`:
```
Successfully installed PyJWT-2.8.0 anyio-4.1.0 asttokens-2.4.1 attrs-21.4.0 beautifulsoup4-4.12.2 bleach-6.1.0 cachetools-5.3.2 certifi-2023.11.17 charset-normalizer-3.3.2 cirq-1.2.0 cirq-aqt-1.2.0 cirq-core-1.2.0 cirq-ft-1.2.0 cirq-google-1.2.0 cirq-ionq-1.2.0 cirq-pasqal-1.2.0 cirq-rigetti-1.2.0 cirq-web-1.2.0 comm-0.2.0 contourpy-1.2.0 cycler-0.12.1 decorator-5.1.1 defusedxml-0.7.1 duet-0.2.9 executing-2.0.1 fastjsonschema-2.19.0 fonttools-4.45.1 google-api-core-2.14.0 google-auth-2.23.4 googleapis-common-protos-1.61.0 grpcio-1.59.3 grpcio-status-1.59.3 h11-0.14.0 httpcore-0.16.3 httpx-0.23.3 idna-3.6 ipython-8.18.1 ipywidgets-8.1.1 iso8601-1.1.0 jedi-0.19.1 jinja2-3.1.2 jsonschema-4.17.3 jupyter-client-8.6.0 jupyter-core-5.5.0 jupyterlab-pygments-0.3.0 jupyterlab-widgets-3.0.9 kiwisolver-1.4.5 lark-0.11.3 markupsafe-2.1.3 matplotlib-3.8.2 matplotlib-inline-0.1.6 mistune-3.0.2 mpmath-1.3.0 msgpack-1.0.7 nbclient-0.9.0 nbconvert-7.11.0 nbformat-5.9.2 networkx-2.8.8 numpy-1.26.2 packaging-23.2 pandas-2.1.3 pandocfilters-1.5.0 parso-0.8.3 pexpect-4.9.0 pillow-10.1.0 platformdirs-4.0.0 prompt-toolkit-3.0.41 proto-plus-1.22.3 protobuf-4.25.1 ptyprocess-0.7.0 pure-eval-0.2.2 py-1.11.0 pyasn1-0.5.1 pyasn1-modules-0.3.0 pydantic-1.10.13 pygments-2.17.2 pyparsing-3.1.1 pyquil-3.2.1 pyrsistent-0.20.0 python-dateutil-2.8.2 python-rapidjson-1.13 pytz-2023.3.post1 pyzmq-25.1.1 qcs-api-client-0.21.6 requests-2.31.0 retry-0.9.2 retrying-1.3.4 rfc3339-6.2 rfc3986-1.5.0 rpcq-3.11.0 rsa-4.9 ruamel.yaml-0.18.5 ruamel.yaml.clib-0.2.8 scipy-1.11.4 six-1.16.0 sniffio-1.3.0 sortedcontainers-2.4.0 soupsieve-2.5 stack-data-0.6.3 sympy-1.12 tinycss2-1.2.1 toml-0.10.2 tornado-6.3.3 tqdm-4.66.1 traitlets-5.14.0 typing-extensions-4.8.0 tzdata-2023.3 urllib3-2.1.0 wcwidth-0.2.12 webencodings-0.5.1 widgetsnbextension-4.0.9
```

Which prints includes: pyquil-3.2.1, msgpack-1.0.7, rpcq-3.11.0

And now this resulted in me finding that there is https://github.com/rigetti/rpcq/commit/801e7a70c69d17ba26a63b4cef32aa78510c8740 which bumps rpcq to 3.11.0 but they did not tag it...

**Comment by BastianZim** [comment]
Ohhhh very good sleuthing! I would suggest that we wait 48h for them to publish a release and if not, we can push something out ourselves with the patch applied. 

Please feel free to add yourself as a maintainer to the rpcq feedstock as well!

**Pull request merged**